urbarium/settlement/models.py

```diff
@@ -1,3 +1,36 @@
 from django.db import models
 
 # Create your models here.
+
+
+class AdministrativeUnit(models.Model):
```

Comment:
Ta class ne spada sem - v Settlement je foreign key na tvorbo oziroma "PoliticalFormationi", ki pa bo v appu z istim imenom

Reply:
Ok, bom izbrisal pa dal ForeignKey

Reply:
Saj s tem lahko zaenkrat še počakaš

Reply:
Aja pa če že o tem piševa, itak bo to večinoma zadev v character in sttlement a ne bo?